xivModdingFramework/General/Enums/XivLanguage.cs

```diff
@@ -33,7 +33,11 @@ public enum XivLanguage
         [Description("de")] German,
         [Description("fr")] French,
         [Description("ko")] Korean,
-        [Description("zh")] Chinese
+        //esrinzou for china ffxiv
```

Comment:
Is there a reason this was changed to "chs"?

The official 2 letter language code for Chinese is "zh", does the game offer different region codes like "chs", "cht", "cn"?

Reply:
The game internally uses chs for Chinese Simplified, and cht for Chinese Traditional.

Reply:
I may have to create 2 different language options for Chinese then

Reply:
ufx is right.